themes/startwords/README.md

```diff
@@ -1,31 +1,28 @@
-# Blank
+# <div align="center"><img src="https://startwords.cdh.princeton.edu/logotype.svg" /> Theme</div>
```

Comment:
Image should have an alt text here (or maybe split out the logo and the theme label?).

Maybe use markdown image syntax? `![Alt](/path/to/img.jpg “image title”)`

Reply:
@rlskoeser I was thinking of simply editing the readme file in the `theme-readme-GW1` branch and then clicking "resolve conversation" for the relevant parts of the conversation in this PR. At least for the elements that I can resolve right now. Does that work?

Reply:
That sounds like a good workflow to me.